webapp/sources/ldap-inventory/inventory-repository/src/main/scala/com/normation/inventory/ldap/core/ReadOnlySoftwareInventoryDAOImpl.scala

```diff
@@ -135,6 +138,8 @@ class ReadOnlySoftwareDAOImpl(
                                  val orFilter = BuildFilter.OR(nodeIds.map(x => EQ(A_NODE_UUID, x.value)): _*)
                                  (con.searchSub(nodeBaseSearch, orFilter, A_SOFTWARE_DN), t2)
                                }
+                               // Wait for the LDAP to answer totally
+                               _            = Thread.sleep(2000L)
```

Comment:
I think there is two things that we can try before adding sleep: 
- I believe that adding a `nodes.toVector` (or `softwareEntries.toVector/toList`) make the request finish
- there's an option in `LDAPConnection` to say that the connection should be synchronous. We could try to set that?
It can be set by a JVM param: `-Dcom.unboundid.ldap.sdk.LDAPConnectionOptions.defaultUseSynchronousMode=true` (and change it in `RudderLdapConnectionOptions` if it works